lambdanotes/src/App.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import './App.css';
+import Sidebar from "./components/sidebar/Sidebar"
```

Comment:
@Raymondgrc the Sidebar folder, under components, is capitalized, so either change to "./components/Sidebar/Sidebar", or lowercase the Sidebar folder to be sidebar. 👍